### PR TITLE
Copy update on /internet-of-things/edgex

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done
- Page refresh on `/internet-of-things/edgex`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]
- Please check page against [copy doc](https://docs.google.com/document/d/17tvfJBC5VncEVWkEsbyW_slAFFSoWV3jQUN7sa8nkOs/edit) and resolve comments
## Issue / Card

Fixes [#4653](https://github.com/canonical-web-and-design/web-squad/issues/4653)

## Screenshot 
EdgeX - Unifying the IoT edge | Ubuntu ()

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Review by sowasred2012 — COMMENTED** [review]
Looking good, just one tweak to the pattern, and also in the copy doc there's:

"You can’t go wrong – literally"

In the page it's "You can’t go wrong. Literally"

**Comment by minkyngkm** [comment]
Thanks @sowasred2012 for reviewing - I have made some changes :)

File: templates/internet-of-things/edgex.html [hunk 1]
```diff
@@ -239,26 +239,99 @@ <h3 class="p-heading--four">Built like cloud-native</h3>
         attrs={"style": "max-height: 120px; width: auto;", "class": "u-sv3"},
         ) | safe
       }}
-      <h3 class="p-heading--four">You can't go wrong. Literally</h3>
+      <h3 class="p-heading--four">You can't go wrong. literally</h3>
```
Comment (sowasred2012) [inline templates/internet-of-things/edgex.html:242]:
```suggestion
      <h3 class="p-heading--four">You can't go wrong &mdash; literally</h3>
```

**Review by sowasred2012 — APPROVED** [review]
One tiny change, and then it's good to go when we get the PDF.

**Pull request merged**